helm/templates/service.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ .Chart.Name }}
   labels:
     release: {{ .Release.Name }}
+  {{- if .Values.service.annotations }}
+  annotations:
```

Comment:
we avoid the conditional with `with` in most places don't we? For example in this chart,
```
    {{- with .Values.imagePullSecrets }}
      imagePullSecrets:
        {{- toYaml . | nindent 8 }}
    {{- end }}
```
Seems slightly cleaner and more consistent with the rest of the chart

Reply:
done